README.md

```diff
@@ -1,52 +1,5 @@
 # HuJSON - Human JSON
 
-The HuJSON decoder is a JSON decoder that also allows
```

Comment:
This grammar is already shown on Go package doc, no need to show it again in the readme.